Dockerfile

```diff
@@ -54,10 +54,6 @@ WORKDIR /tmp/nemo
 COPY requirements .
 RUN for f in $(ls requirements*.txt); do pip3 install --disable-pip-version-check --no-cache-dir -r $f; done
 
-# install pynini
-COPY nemo_text_processing/install_pynini.sh /tmp/nemo/
-RUN /bin/bash /tmp/nemo/install_pynini.sh
```

Comment:
Should we modify the NeMo README.rst to get rid of the "install_pynini.sh" instructions, and include instructions for installing nemo_text_processing?

https://github.com/NVIDIA/NeMo/blob/main/README.rst#nemo-text-processing

Reply:
updated README